force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.js

```diff
@@ -132,4 +133,22 @@ export default class NewServiceSchedule extends LightningElement {
             ? event.detail[0].value
             : undefined;
     }
+
+    handleStartChange(event) {
```

Comment:
This makes it so much more clear in the UI with the defaults, love this addition!

Reply:
We might need a little bullet proofing though. I get an error if I click end and then click start...
![EndBeforeStart](https://user-images.githubusercontent.com/450473/94022856-2615f780-fd7b-11ea-855f-1c8266f52a0b.png)